src/robot/running/testlibraries.py

```diff
@@ -91,13 +91,17 @@ def doc(self):
         return self._doc
 
     @property
-    def listener(self):
+    def listeners(self):
         if self.has_listener:
-            return self._get_listener(self.get_instance())
+            return self._get_listeners(self.get_instance())
         return None
 
-    def _get_listener(self, inst):
-        return getattr(inst, 'ROBOT_LIBRARY_LISTENER', None)
+    def _get_listeners(self, inst):
+        try:
+            listeners = getattr(inst, 'ROBOT_LIBRARY_LISTENER')
+            return listeners if isinstance(listeners, list) else [listeners]
```

Comment:
Testing just against Python list has a problem that Java libraries cannot register a list of listeners. We have `utils.is_list_like` that could be used here instead, but it has a potential problem that a listener itself could be iterable. I guess iterable listeners are rare enough that this isn't a big problem. If someone actually faces that problem, they can wrap the iterable listener into a list.